_posts/2021-03-10-application-modernisation-101.markdown

```diff
@@ -0,0 +1,74 @@
+
+## Investment versus ROI
+Regarding the tradeoff between initial investment and return, there’s a clear relationship:  
+![More upfront = more return](/img/blog/appmod101/roi-graph.png)
+
+It’s easy to see that embarking on a rehosting exercise of moving all your applications and services to VM’s in the cloud is a quick and cheap exercise, but as the graph shows, the costs with this approach won’t end when the instances are powered up. Overall, a one for one swap rarely works out cost effective, and none of the benefits of scalability, availability and resilience are gained. If you have the time, smart decision makers are opting for the long game, and really leveraging off the benefits of rearchitecting, refactoring and rebuilding for cloud native deployments.
+
+## Pros and Cons
+
+
+
+|                       |Rehost|Replatform|Refactor/Rebuild|
+|-----------------------|------|----------|----------------|
+|Upfront Cost|Low|Medium/High|High|
+|Ongoing Cost|High|Medium|Low|
+|Implementation Timeline|Short|Medium|Long|
+|Complexity|Low|Medium|High|
+|Scalability|Low|High|High|
+|Fault Tolerance|Medium|High|High|
+|Availability|Medium|High|High|
+|Skills availability|High|Low|Medium|
+
+** Please note, these categorisations are indicative and relative that depend completely on the Application/Workload. There are a multitude of exceptions to these rules of thumb.
+
+## Conclusion
+The landscape of digital infrastructure is changing - your challenge is to decide how to leverage it for the benefit of your organisation There’s no one right way to move your applications into 2021. The only failure would be to not make a decision at all. 
+
+The approach you take will vary depending on your individual circumstances, your budget and your timeline. Fortunately, there is a spectrum of solutions available, each with their own pros and cons. The real benefits of cloud are only fully realised as you trend towards the refactor/rebuild end of the spectrum. Hopefully this article provides some insight as to what your options are. Stay tuned to our blog for an in-depth look into replatforming, coming soon.
```

Comment:
want to throw in this link at the end as a CTA?

https://www.mechanicalrock.io/our-expertise/application-modernisation